examples/3d/motion_blur.rs

```diff
@@ -304,12 +305,12 @@ fn race_track_pos(offset: f32, t: f32) -> Vec2 {
     let x_tweak = 2.0;
     let y_tweak = 3.0;
     let scale = 8.0;
-    let x0 = (x_tweak * t).sin();
-    let y0 = (y_tweak * t).cos();
-    let dx = x_tweak * (x_tweak * t).cos();
-    let dy = y_tweak * -(y_tweak * t).sin();
-    let x = x0 + offset * dy / (dx.powi(2) + dy.powi(2)).sqrt();
-    let y = y0 - offset * dx / (dx.powi(2) + dy.powi(2)).sqrt();
+    let x0 = sin(x_tweak * t);
+    let y0 = cos(y_tweak * t);
+    let dx = x_tweak * cos(x_tweak * t);
+    let dy = y_tweak * -sin(y_tweak * t);
+    let x = x0 + offset * dy / (FloatPow::squared(dx) + FloatPow::squared(dy)).sqrt();
+    let y = y0 - offset * dx / (FloatPow::squared(dx) + FloatPow::squared(dy)).sqrt();
```

Comment:
Again, should be refactorable to `hypot`.
```suggestion
    let dl = hypot(dx, dy);
    let x = x0 + offset * dy / dl;
    let y = y0 - offset * dx / dl;
```